mappings/net/minecraft/block/AbstractBlock.mapping

```diff
@@ -9,19 +9,27 @@ CLASS net/minecraft/class_4970 net/minecraft/block/AbstractBlock
 	FIELD field_23161 randomTicks Z
 	FIELD field_23162 soundGroup Lnet/minecraft/class_2498;
 	FIELD field_23163 slipperiness F
+	FIELD field_23164 velocityMultiplier F
 	FIELD field_23165 slowDownMultiplier F
 	METHOD <init> (Lnet/minecraft/class_4970$class_2251;)V
 		ARG 1 settings
 	METHOD method_16841 getOffsetType ()Lnet/minecraft/class_4970$class_2250;
 	METHOD method_17454 createScreenHandlerFactory (Lnet/minecraft/class_2680;Lnet/minecraft/class_1937;Lnet/minecraft/class_2338;)Lnet/minecraft/class_3908;
 	METHOD method_19286 onProjectileHit (Lnet/minecraft/class_1937;Lnet/minecraft/class_2680;Lnet/minecraft/class_3965;Lnet/minecraft/class_1676;)V
 	METHOD method_22358 canBucketPlace (Lnet/minecraft/class_2680;Lnet/minecraft/class_3611;)Z
+	METHOD method_25959 getSidesShape (Lnet/minecraft/class_2680;Lnet/minecraft/class_1922;Lnet/minecraft/class_2338;)Lnet/minecraft/class_265;
+	METHOD method_26159 getVisualShape (Lnet/minecraft/class_2680;Lnet/minecraft/class_1922;Lnet/minecraft/class_2338;Lnet/minecraft/class_3726;)Lnet/minecraft/class_265;
```

Comment:
Hmm, mind restore the parameters?

Reply:
Oh right, params.

Reply:
Actually, what parameters do you want me to restore?

Reply:
Hmm, somehow all the parameter names are lost... Like `world`, `pos`, etc

Reply:
Ah yeah, I can fix that real quick.

Reply:
all the parameters are in!